### ISSUE TITLE
WCAG-EM Report Tool

### ISSUE THREAD

**Issue by nitedog** [w3c/EasyChecks#48]
[high] Please add a sub-bullet under WCAG-EM, to refer people to the WCAG-EM Report Tool (https://www.w3.org/WAI/eval/report-tool/). It can be very helpful for this audience to know right away that there is practical tool to guide them through the WCAG-EM specification.

**Comment by slhenry** [comment]
I'm not convinced this is worth the clutter. The WCAG-EM link goes to the [overview page,](https://www.w3.org/WAI/eval/conformance) and the Report Tool is listed on that page. Perhaps the Report Tool should be more prominent and earlier on that page?

**Comment by bakkenb** [comment]
I understand what you are suggesting @nitedog. My first thought was that I would rather people see the link to EM and read it all *before* going to look at the EM Report Tool. If we put the link to the Report Tool in this list, they may focus on that and not ever even look at EM and not have a good understanding of the methodology when using the tool. 

My second thought was that the link to the EM Report Tool is pretty far down on the EM page. Then I read that @slhenry thought the same thing. So maybe that should be considered for moving further up the page.

My [medium] preference would be to not place the link for the EM Report Tool in the bulleted list.

**Comment by slhenry** [comment]
Resolved in [16 Dec 2016 telecon](https://www.w3.org/2016/12/16-eo-minutes.html#item06)
